.travis.yml

```diff
@@ -36,9 +36,8 @@ before_install:
 install:
   - node --version
   - npm --version
-  - npm install electron-packager
-  - npm install
-  - npm run release
+  - npm install && cd app && npm install && cd ..  # Hack until instapp-app-deps works on travis
```

Comment:
Not sure why the github PR review tool is showing that you added these files in your branch... These changes already exist in master and were added by another branch.

Did you merge in master into your branch? Or just copy/paste the files in master into your branch..  A lot of these diffs are unnecessary and not a part of this branch and have already been reviewed/approved... 

It's cluttering that they're showing up in this PR